bot/exts/halloween/hacktober-issue-finder.py

```diff
@@ -59,40 +59,39 @@ async def get_issues(self, ctx: commands.Context, option: str) -> Optional[Dict]
             log.debug("using cache")
             return self.cache_normal
 
-        async with aiohttp.ClientSession() as session:
+        if option == "beginner":
+            url = URL + '+label:"good first issue"'
+            if self.cache_beginner is not None:
+                page = random.randint(1, min(1000, self.cache_beginner["total_count"]) // 100)
+                url += f"&page={page}"
+        else:
+            url = URL
+            if self.cache_normal is not None:
+                page = random.randint(1, min(1000, self.cache_normal["total_count"]) // 100)
+                url += f"&page={page}"
+
+        log.debug(f"making api request to url: {url}")
+        async with self.bot.http_session.get(url, headers=REQUEST_HEADERS) as response:
+            if response.status != 200:
+                log.error(f"expected 200 status (got {response.status}) from the GitHub api.")
+                await ctx.send(f"ERROR: expected 200 status (got {response.status}) from the GitHub api.")
```

Comment:
I think we can merge the text here and just send one message; and just do a `return` instead of explicitly `return None` to stay consistent with how it's used in the rest of the codebase

Reply:
I agree with the first part, but if the function is meant to return something, so we should explicitly return `None`,

Reply:
Resolved in https://github.com/python-discord/sir-lancebot/pull/718/commits/19e2efaf5572a0edb4595bead6877aaae078adae.